metadata.json

```
@@ -63,7 +63,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 6.0.0"
+      "version_requirement": ">= 4.19 < 6.0.0"
```

please also provide the patch level:
```suggestion
      "version_requirement": ">= 4.19.0 < 6.0.0"
```

---

stdlib 6.0.0 was released 5 hours ago.

---

```suggestion
      "version_requirement": ">= 4.19.0 < 7.0.0"
```

---

Changed as requested.